doc/modules/tree.rst

```diff
@@ -515,9 +515,9 @@ Mean Absolute Error:
 
 .. math::
 
-    \bar{y}_m = \frac{1}{N_m} \sum_{i \in N_m} y_i
+    median(y)_m = median\{y_i for i in N_m\}
```

Comment:
Currently looks like this:
<img width="343" alt="Screen Shot 2020-03-18 at 1 43 14 PM" src="https://user-images.githubusercontent.com/5402633/76990552-82a4df80-691e-11ea-9089-5d6458645113.png">

Reply:
Better now?